src/io/flutter/dart/DartPlugin.java

```diff
@@ -18,7 +18,7 @@
   /**
    * Tracks the minimum required Dart Plugin version.
    */
-  private static final String MINIMUM_REQUIRED_PLUGIN_VERSION = "162.2485";
+  private static final String MINIMUM_REQUIRED_PLUGIN_VERSION = "163.10154";
```

Comment:
Note that this is the bound @alexander-doroshko suggested.  It could probably just as well be `163.11306`.